test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -124,13 +124,21 @@ string TestFunctionCall::formatBytesParameters(bytes const& _bytes, dev::solidit
 	stringstream resultStream;
 	if (_bytes.empty())
 		return {};
+	auto sizeFold = [](size_t _a, Parameter const& _b) { return std::move(_a) + _b.abiType.size; };
```

Comment:
```suggestion
	auto sizeFold = [](size_t _a, Parameter const& _b) { return _a + _b.abiType.size; };
```
No point in moving ``size_t`` :-).

Reply:
Fxed :)